data/news/platform/ocaml-5.0.rc1.md

```diff
@@ -0,0 +1,79 @@
+---
+title: OCaml 5.0.0 - First release candidate
+description: First release candidate of OCaml 5.0.0
+date: "2022-12-06"
+tags: [ocaml, platform]
+---
+
+
+The release of OCaml version 5.0.0 is imminent. As a final step before the
```

Comment:
```suggestion
The release of OCaml version 5.0 is imminent. As a final step before the
```
I think I read that we’re going with 5.0 instead of 5.0.0 for this first release.

Reply:
The release number will be 5.0.0 for the first release. I think it is a good idea to abbreviate it to 5.0 whenever we are speaking of an OCaml version as seen by users, but I will rather keep the full release number for the release announce themselves.